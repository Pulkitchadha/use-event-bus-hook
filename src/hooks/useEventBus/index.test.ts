import { renderHook } from '@testing-library/react-hooks';
import { useEventBus } from '.';

describe("useFetchEmployee hook", () => {
    it("Should be defined", () => {
        expect(useEventBus).toBeDefined();
    });

    it("Should match the snapshot", () => {
        const handler = jest.fn();
        const { result } = renderHook(() => useEventBus("Event_Name", handler));
        expect(result).toMatchSnapshot();
    });
});