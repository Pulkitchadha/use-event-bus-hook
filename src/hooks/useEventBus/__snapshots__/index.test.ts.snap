// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`useFetchEmployee hook Should match the snapshot 1`] = `
Object {
  "all": Array [
    undefined,
  ],
  "current": undefined,
  "error": undefined,
}
`;
